### PR TITLE
Bug 1201137 - Give visual feedback when dragging icons. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ee24e0a6c0292809914dfece90e52e1f0d51cf8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=894d20797f301327fef3f175a2024800e649f84f

File: apps/homescreen/style/apps-panel.css [hunk 1]
```diff
@@ -64,6 +64,14 @@ gaia-container.small > .gaia-container-child.dragging > .icon-container > gaia-a
   transform: scale(0.8) translate(-3.2vw, -3.2vw);
 }
 
+gaia-container > .gaia-container-child > .icon-container.hover-before {
+  transform: translateX(3.5rem);
+}
+
+gaia-container > .gaia-container-child > .icon-container.hover-after {
+  transform: translateX(-3.5rem);
```
Comment (gmarty) [inline apps/homescreen/style/apps-panel.css]: Is this value purposely the same for the 3 or 4 icons layout?

Comment (Cwiiis) [inline apps/homescreen/style/apps-panel.css]: I found it looks about right in both that it wasn't worth having a separate value... I expect UX to come back at some point in the future and disagree :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=33a0c40ed57d98abf089303d8c6dfa1accc2ca9c

**Pull request merged**